src/App.tsx

```diff
@@ -20,17 +29,138 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    step: 3,
+    frameSize: 3,
+    itemWidth: 130,
+    animationDuration: 1,
+    infinite: false,
+  };
+
+  stepChangeHandler = (event: ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value === ''
+      || Number(event.target.value) < 1
+      || Number.isNaN(Number(event.target.value))) {
+      this.setState({
+        step: 1,
+      });
+    }
+
+    this.setState({
+      step: Number(event.target.value),
+    });
+  };
+
+  frameSizeChangeHandler = (event: ChangeEvent<HTMLInputElement>) => {
+    if (
+      event.target.value === ''
+      || Number(event.target.value) < 1
+      || Number.isNaN(Number(event.target.value))
+    ) {
+      this.setState({
+        frameSize: 1,
+      });
+    }
+
+    this.setState({
+      frameSize: Number(event.target.value),
+    });
```

Comment:
you could also think of this approach - unify all the numeric inputs change handler - so you only have just one input and that's it
```suggestion
    const { name, value } = event.target;

    this.setState({
      [name]: Number(value),
    });
```

to make this work you'll need to provide each Input with a name attribute that corresponds with the state names.
```
<input type="number" name="frameSize" ...
// and so on
```